training/culture-and-process/PR_REVIEW_AND_RESPONSE_GUIDANCE.md

```diff
@@ -0,0 +1,121 @@
+Pull Request Review and Response Guidance
+=========================================
+
+General guidance on reviewing a pull request and responding
+
+### Prerequisites, Philosophy and Offering feedback
+
+When you are making an observation or giving helpful feedback for a specific piece of code (or a general overall comment), if what you are writing is over ~50 words - it may be better to have a direct call or discuss via Slack and then maybe a comment can be placed in the code as to why something is done a particular way.
+
+**Teamwork** - Reviewing other peoples work is an act of teamwork and teamwork can be challenging as it requires compromise.
+
+**Reviews** - Are an opportunity for learning - so don't just say 'do this' - consider explaining why, especially when its a best practice. Try and explain the value (maybe with links to articles), but be careful that the links you reference are balanced.
+
+**Kiss** - Keep It Simple and Straightforward. Does the body of work being reviewed lend itself to Maintenance ? Where applicable, are there sufficient comments explaining why and how something is being done.
```

Comment:
The uppercase M in "Maintenance" should be lowercase.

Reply:
Done

Reply:
done